providers/src/airflow/providers/edge/worker_api/routes/_v2_routes.py

```diff
@@ -20,12 +20,15 @@
 
 import json
 import logging
-from functools import cache
-from typing import TYPE_CHECKING, Any, Callable
+from typing import TYPE_CHECKING, Any
 from uuid import uuid4
 
 from flask import Response, request
 
+from airflow.api_internal.endpoints.rpc_api_endpoint import (  # type: ignore[attr-defined]
+    # Note: This is just for compatibility with Airflow 2.10, not working for Airflow 3 / main as removed
+    initialize_method_map,
```

Comment:
@potiuk FYI in my cleanup I realized... as the internal API is "gone" (virtually, at least not be usable, will contribute in a moment to clean as well...) and as workaround a copy was made... actually that part of code is only working in Airflow 2.10. Therefore removing the copy of the full method map here in the cleanup and sourcing this from the core (=2.10)

This way the "traces" are reduced, just a import from the past is used until we drop 2.10 support in Edge.